### PR TITLE
Creating checkbox component

### PR DESCRIPTION
**Pull request by luciejachimova**

### CODE REVIEW

File: src/components/checkbox/checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+interface CheckboxProps {
+    label?: React.ReactNode
```
Comment (Mystypix) [inline src/components/checkbox/checkbox.tsx:4]: I would make this as required. Not sure if we want checkbox without label :)

Comment (Mystypix) [inline src/components/checkbox/checkbox.tsx:4]: I would make this as required. Not sure if we want checkbox without label :)

File: src/components/checkbox/checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+export const Checkbox = ({
+    label,
+    register,
+    name,
+    defaultValue,
+    ...rest
+}: CheckboxProps) => {
+    return (
+        <div className="checkbox-wrapper">
```
Comment (Mystypix) [inline src/components/checkbox/checkbox.tsx:17]: We are using css modules :) please check how to use them https://css-tricks.com/css-modules-part-1-need/
TLDR - you need to import css like this `import css from './name-of-the-file.module.scss` and use `css` in class with dot notation like this `className={css.checkboxWrapper}`

File: src/components/checkbox/checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            </label>
+            <input
+                type="checkbox"
+                id={name}
+                name={name}
+                defaultChecked={defaultValue}
+                {...register(name)}
+                {...rest}
+            />
+            <span className="checkmark"></span>
```
Comment (Mystypix) [inline src/components/checkbox/checkbox.tsx:29]: I've checked your code in application and you probably forgot to hide actual input :) or i'm not sure what you wanted to achieve here
